packit/cli/prepare_sources.py

```diff
@@ -0,0 +1,108 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+import logging
+import os
+
+import click
+
+from packit.cli.types import LocalProjectParameter
+from packit.cli.utils import cover_packit_exception, get_packit_api
+from packit.config import pass_config, get_context_settings
+from packit.schema import JobConfigSchema
+
+logger = logging.getLogger("packit")
+
+
+@click.command("prepare-sources", context_settings=get_context_settings())
+@click.option(
+    "--result-dir",
+    metavar="DIR",
+    help="Copy the sources into DIR.",
+)
+@click.option(
+    "--upstream-ref",
+    default=None,
+    help="Git ref of the last upstream commit in the current branch "
+    "from which packit should generate patches "
+    "(this option implies the repository is source-git).",
+)
+@click.option(
+    "--bump/--no-bump",
+    default=True,
+    help="Specifies whether to bump version or not.",
+)
+@click.option(
+    "--release-suffix",
+    default=None,
+    type=click.STRING,
+    help="Specifies release suffix. Allows to override default generated:"
+    "{current_time}.{sanitized_current_branch}{git_desc_suffix}",
+)
+@click.option(
+    "--job-config",
+    default=None,
+    type=click.STRING,
+    help="Overrides package config found in the repository.",
```

Comment:
I'm sorry but I don't understand this explanation.

Reply:
in that case I'd hide this

the other options look relevant to me even for our users (and us testing things out)